4_team_page/src/data/Team.js

```diff
@@ -0,0 +1,104 @@
+import IMAGES from "../assets/member_images/images.js";
+
+export const Team = [
+	{
+		"id": 101,
+		"img": IMAGES.upaharika,
```

Comment:
Consider IMAGES as coming from API data. You can't use constants there. Use constants for labels etc., static data of your site.

Reply:
Agreed. Since the data was static here, found it safe to use const. Changed.